toolkits/nature/packages/nature-section-heading/scss/10-settings/section-heading.scss

```diff
@@ -1,6 +1,6 @@
 $section-heading--border-top: 5px solid black;
 $section-heading--border-bottom: 1px solid black;
-$section-heading--padding: spacing(8) 0;
+$section-heading--padding: spacing(8);
```

Comment:
I'd go one further here, being a little more explicit and ready for future changes and declare them as 

```
$section-heading--padding--top: spacing(8);
$section-heading--padding--bottom: spacing(8);
```